make-travis-yml.cabal

```diff
@@ -104,7 +104,9 @@ test-suite golden
                      , filepath
                      , directory
                      , bytestring
-                     , tasty >= 0.11.3 && <0.12
+                     , tasty >= 0.11.3 && <1.1
                      , tasty-golden >=2.3.1.1 && <2.4
                      , Diff >=0.3.4 && <0.4
   default-language:    Haskell2010
+  if impl(ghc >= 8.4)
```

Comment:
if this is dependency failure please open an issue in https://github.com/haskell-infra/hackage-trustees (and at your discretion at package's own issue tracker).

Reply:
@hvr the new commit fixes it, yes